source/_integrations/heos.markdown

```diff
@@ -63,6 +64,38 @@ Use the sign-in service to sign the connected controller into a HEOS account so
 
 Use the sign-out service to sign the connected controller out of a HEOS account. An error message is logged if sign-out is unsuccessful. There are no parameters to this service.
 
+### Service `heos.join_group`
+
+This service can be used for joining or creating a HEOS group. A HEOS group always contains exactly one leader player and at least another player which acts as a group member. All member players will synchronously play the same audio as the group leader. The group can be controlled by any player of the group. For creating a new HEOS group or joining more members to an existing group the service call looks as follows:
+
+```json
+{
+  "leader": "media_player.heos_player1",
+  "members": [
+    "media_player.heos_player2"
+  ]
+}
+```
+
+| Attribute              | Description
+| ---------------------- | ---------------------------------------------------------------- |
+| `leader`               | The HEOS group leader. [Required]
```

Comment:
Add a column to indicate if an attribute is required or not. Please see other parts of our documentation for an example.